plugins/manual_classification/manual_classification_templates/manual_classification_template.html

```diff
@@ -1,25 +1,66 @@
 {% import 'forms.html' as forms %}
-<style type="text/css">
-    @import url("{{url_for('static', filename='forms.css')}}");
-</style>
-
-<h2>Manual Classification</h2>
-
-{% call forms.render_form() %}
-<!-- class name -->
-{{ forms.render_fields(schema, values=values, errors=errors) }}
-<div class="qhana-form" style="flex-direction: row; flex-wrap: wrap;">
-    <!-- select id's -->
-    {% for id in id_list %}
-    <div class="qhana-form-field" style="width: 3em; height: 2em; padding: 1em;">
-        <input type="checkbox" id={{id}} name={{id}}>
-        <label class="qhana-form-label" for="{{id}}">{{id}}</label>
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='forms.css')}}" media="screen" />
+    <style type="text/css">
+        @import url("{{url_for('static', filename='forms.css')}}");
+
+        table {
+            font-family: arial, sans-serif;
+            border-collapse: collapse;
+            width: 100%;
+        }
+
+        td,
+        th {
+            border: 1px solid #dddddd;
+            text-align: left;
+            padding: 8px;
+        }
+
+        tr:nth-child(even) {
+            background-color: #dddddd;
+        }
+    </style><!-- TODO -->
+</head>
+
+<body>
+    <div class="qhana-microfrontend">
+        {% call forms.render_form() %}
+        <!-- class name -->
+        {{ forms.render_fields(schema, values=values, errors=errors) }}
+        <div class="qhana-form" style="flex-direction: row; flex-wrap: wrap;">
+            <!-- select id's -->
+            <table>
+                <tr>
+                    <th>Select</th>
+                    {% for attr in attr_list%}
+                    <th>{{attr}}</th>
+                    {% endfor %}
+                </tr>
+                {% for entity in entity_list %}
+                <tr>
+                    <td class="qhana-form-field" style="width: 3em; padding: 1em;">
+                        <input type="checkbox" id={{entity["ID"]}} name={{entity["ID"]}}>
+                    </td>
+                    {% for attr in attr_list%}
+                    <td>{{entity[attr] if attr in entity.keys() else ""}}</td>
+                    {% endfor %}
+                </tr>
+                {% endfor %}
+            </table>
+
+        </div>
+        <div class="qhana-form-buttons">
+            {{ forms.submit("validate")}}
+            {{ forms.submit("continue", action=process)}}
+            {{ forms.submit("done", action=done)}}
```

Comment:
We really need to specify how repeatable substeps should work...

Reply:
Should we include the above way in the spec or is it still open for discussion?

Reply:
For now we could include it in the simple template (gated by an if that checks if there is a done URL defined). The micro frontend can find the done URL by internal information. But for other API clients we need to specify how the done URL can be found (without scraping the micro frontend). There are some plans but nothing final, so no spec changes for now. (But you culd start an ADR to sketch some solutions)

Reply:
We could also include the above functionality in simple template in case "done" is specified

Reply:
Yes. See the first sentence of my previous reply.